packages/unigraph-dev-backend/src/custom.d.ts

```diff
@@ -11,4 +11,58 @@ declare global {
 declare type UnigraphUpsert = {
   queries: string[],
   mutations: any[]
-}
\ No newline at end of file
+}
+
+declare type EventCreateUnigraphSchema = {
+  "type": "event",
+  "event": "create_unigraph_schema",
+  schema: object | object[],
+  id: number
+}
+
+declare type EventCreateUnigraphObject = {
+  "type": "event",
+  "event": "create_unigraph_object",
+  object: object,
+  id: number,
+  schema: string | undefined,
+  padding: boolean | undefined
+}
+
+declare type EventDropAll = {
+  type: "event",
+  event: "drop_all",
+  id: number
+}
+
+declare type EventDropData = {
+  type: "event",
+  event: "drop_data",
+  id: number
+}
+
+declare type EventCreateDataByJson = {
+  type: "event",
+  event: "create_data_by_json",
+  id: number,
+  data: object
+}
+
+declare type EventSetDgraphSchema = {
+  type: "event",
+  event: "set_dgraph_schema",
+  id: number,
+  schema: string
+}
+
+declare type EventQueryByStringWithVars = {
+  type: "event",
+  event: "query_by_string_with_vars",
+  vars: object,
+  query: string,
+  id: number
+}
+
+declare interface IWebsocket {
+  send: Function
+}
```

Comment:
you should be able to use the built-in `WebSocket` type for this

Reply:
I couldn't seem to find a type declaration of `Websocket` - but with that said, I haven't used that much of typescript for websocket, could you take a look?